modules/AuraTracker.lua

```diff
@@ -69,7 +69,10 @@ TMW_ST.UnitAuras = {
 		else
 			return getUnitAura(unit, spell)
 		end
-	end
+	end,
+  getUnitAuras = function(unit, spell, onlyMine)
```

Comment:
just add the flag to the original method. there's no reason to create a new method for it (especially since the naming is confusing since both methods return the exact same value if an aura is found so there's no plural)